ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineNewPayload.java

```diff
@@ -178,6 +180,20 @@ public JsonRpcResponse syncResponse(final JsonRpcRequestContext requestContext)
               .map(Bytes::fromHexString)
               .map(in -> TransactionDecoder.decodeOpaqueBytes(in, EncodingContext.BLOCK_BODY))
               .collect(Collectors.toList());
+      transactions.forEach(
+          transaction ->
+              mergeCoordinator
+                  .getEthScheduler()
+                  .scheduleFutureTask(
+                      () -> {
+                        Address sender = transaction.getSender();
+                        LOG.atTrace()
+                            .setMessage("The sender for transaction {} is calculated : {}")
+                            .addArgument(transaction.getHash())
+                            .addArgument(sender)
+                            .log();
+                      },
+                      Duration.ofSeconds(1)));
```

Comment:
This way the execution will start 1 seconds later, are you sure this is what you want?
Could be better to use `scheduleTxWorkerTask` or `scheduleServiceTask`?

Reply:
Nice catch, no I didn't want to delay the execution, I changed to scheduleTxWorkerTask that looks more convenient in this case.